command/src/main/java/pw/chew/jdachewtils/command/OptionHelper.java

```diff
@@ -0,0 +1,169 @@
+package pw.chew.jdachewtils.command;
+
+import net.dv8tion.jda.api.entities.GuildChannel;
+import net.dv8tion.jda.api.entities.IMentionable;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.MessageChannel;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+/**
+ * A collection of useful methods for working with Options.
+ */
+public class OptionHelper {
```

Comment:
It would be good if the class has `final` keyword and, a private constructor to avoid the initiating the class.

Reply:
Good idea, added 👍